pgjdbc-osgi-test/src/test/java/org/postgresql/test/osgi/DataSourceFactoryTest.java

```diff
@@ -120,7 +120,7 @@ private String getUrl() {
         + p.get("server") + ":"
         + p.get("port") + "/"
         + p.get("database")
-        + "?loglevel=" + p.get("loglevel")
+        + "?loggerLevel=" + p.get("loggerLevel")
```

Comment:
loggerLevel has been removed as a property.

Reply:
"loggerLevel" is registered in PgProperty.java
https://github.com/pgjdbc/pgjdbc/blob/master/pgjdbc/src/main/java/org/postgresql/PGProperty.java#L331-L336

Anyway, I just fixed invalid value. "loglevel" is not valid value. It is not described in PGProperty.java
"loggerLevel" is valid value in PGProperty.java and tests does not fail anymore.